packages/renderer/src/lib/container/ContainerList.svelte

```diff
@@ -467,16 +467,22 @@ function setStoppedFilter() {
   </svelte:fragment>
   <svelte:fragment slot="bottom-additional-actions">
     {#if selectedItemsNumber > 0}
-      <Button
-        on:click="{() => deleteSelectedContainers()}"
-        aria-label="Delete selected containers and pods"
-        title="Delete {selectedItemsNumber} selected items"
-        bind:inProgress="{bulkDeleteInProgress}"
-        icon="{faTrash}" />
-      <Button
-        on:click="{() => createPodFromContainers()}"
-        title="Create Pod with {selectedItemsNumber} selected items"
-        icon="{SolidPodIcon}" />
+      <div style="display: inline-flex;">
```

Comment:
Hello, if you want to apply style, as we're using tailwind for CSS you should use `inline-flex`
```suggestion
      <div class="inline-flex">
```

https://tailwindcss.com/docs/display